xbmc/addons/kodi-addon-dev-kit/Makefile

```diff
@@ -0,0 +1,30 @@
+ifeq ($(CMAKE),)
+  CMAKE = cmake
+endif
+
+ifeq ($(PREFIX),)
+  PREFIX = /usr/local
```

Comment:
shouldn't be needed after adding the include from above.
That is assuming addon-dev-kit is always built with kodi and after configure